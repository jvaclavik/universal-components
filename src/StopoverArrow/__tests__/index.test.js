// @flow

import * as React from 'react';
import { shallow, render } from 'react-native-testing-library';
import { StopoverArrow } from '..';

describe('StopoverArrow', () => {
  const stops = 3;
  const { getByProps } = render(<StopoverArrow stops={stops} />);

  it('should have correct props', () => {
    expect(getByProps({ stops })).toBeDefined();
  });

  it('should match the snapshot', () => {
    const { output } = shallow(getByProps({ stops }));
    expect(output).toMatchSnapshot();
  });
});
