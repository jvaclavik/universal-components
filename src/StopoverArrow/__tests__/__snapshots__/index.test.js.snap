// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StopoverArrow should match the snapshot 1`] = `
<View>
  <Svg
    fill="none"
    height="9"
    viewBox="0 0 53 9"
    width="53"
    xmlns="http://www.w3.org/2000/svg"
  >
    <React.Fragment>
      <Rect
        fill="#7f91a8"
        height="1"
        width="49"
        x="2"
        y="4.00024"
      />
      <Rect
        fill="#7f91a8"
        height="5"
        transform="rotate(45 51.3281 3.79321)"
        width="1"
        x="51.3281"
        y="3.79321"
      />
      <Rect
        fill="#7f91a8"
        height="1"
        transform="matrix(-0.707107 -0.707107 -0.707107 0.707107 52.0354 4.50049)"
        width="5"
      />
    </React.Fragment>
    <Rect
      fill="#e8edf1"
      height="4"
      stroke="#7f91a8"
      width="4"
      x={14}
      y="3"
    />
    <Rect
      fill="#e8edf1"
      height="4"
      stroke="#7f91a8"
      width="4"
      x={22}
      y="3"
    />
    <Rect
      fill="#e8edf1"
      height="4"
      stroke="#7f91a8"
      width="4"
      x={30}
      y="3"
    />
  </Svg>
</View>
`;
